examples/run_script_with_local_browser/run_script_with_local_browser.py

```diff
@@ -0,0 +1,77 @@
+"""This example demonstrates how to run the script in an existing local browser with AgentQL."""
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# The URL to connect to your local existing browser.
+WEBSOCKET_URL = "YOUR_WEBSOCKET_URL"
+
+URL = "https://scrapeme.live/shop"
+
+QUERY = """
```

Comment:
If we expect a specific page to run this query, let's add instructions to open this page in README